package.json

```diff
@@ -20,6 +20,7 @@
     "prebuild:mobile": "npm run clean:mobile",
     "prewatch:mobile": "npm run clean:mobile",
     "lint": "npm-run-all --parallel 'lint:*'",
+    "lint:js:modified": "eslint --fix $(git ls-files -m | grep -E 'jsx?$')",
```

Comment:
What about a precommit hook using [lint-staged](https://github.com/okonet/lint-staged)? Still skippable via `--no-verify` for cases in which we want to commit anyway.

Reply:
True I forgot about lint-staged. I'm removing this from the PR.